lessons/beginners/str/index.md

```diff
@@ -289,8 +371,6 @@ vypis = 'Ahoj {jmeno}! Výsledek je {cislo}.'.format(cislo=7, jmeno='Elvíro')
 print(vypis)
 ```
 
-Formátování se používá skoro všude, kde je
-potřeba „hezky“ vypsat nějakou hodnotu.
```

Comment:
tohle je pryč schválně?

Reply:
schválně, ale ne úplně chtěně 
ony ty f-stringy z Brna končí "Ale nepřežeň to! Většinou je program přehlednější, když si každou vypisovanou hodnotu zvlášť pojmenuješ." 
Tak se mi tam tahle věta k tomu nehodila. 

Ale asi je možno jí strčit nakonec, za tu metodu format. (tam vlastně asi i původně byla)

Reply:
Takže v Brně tu větu nemají? Vidím, že to @encukou vzal z gruntu https://github.com/encukou/naucse.python.cz/commit/ef80d3c6f318a3f3807cf594f0c0d1255f7d5041

Reply:
V Brně tuhle větu nemají. Oni navíc mají na formátování řetězců vlastní sekci a funkci format mají v sekci Řetězcové fce a metody. 
Ale přijde mi, že to patří k sobě, jen to první chceme ukázat a na to druhé spíš jen upozornit, že to můžou potkat, tak aby věděli, o co jde.

Reply:
Strčíš jí nakonec teda?

Reply:
OK